includes/WebRequest.php

```diff
@@ -1134,6 +1134,23 @@ public function isValidWriteRequest( \User $user ) {
 	public function isWikiaInternalRequest() {
 		return $this->getHeader( self::WIKIA_INTERNAL_REQUEST_HEADER ) !== false;
 	}
+
```

Comment:
Eh, I think we can let that practice go.  Before we forked we added those lines so that it would make it easier to merge newer versions of MW into our highly customized build.  We kept it up for about a year after that just in case we needed to merge again, however now it just adds noise and no real value.